lib/main.js

```diff
@@ -11,9 +11,9 @@ const {
   removeUnusedImportProvider
 } = actionProviders
 
-const serverDownloadUrl = 'http://download.eclipse.org/jdtls/milestones/0.14.0/jdt-language-server-0.14.0-201802282111.tar.gz'
+const serverDownloadUrl = 'http://download.eclipse.org/jdtls/milestones/0.34.0/jdt-language-server-0.34.0-201902280418.tar.gz'
 const serverDownloadSize = 35873467
```

Comment:
I believe this also needs to changed, right?

Reply:
Not sure if this is checked at all. I just changed these two lines in my local copy, restarted Atom, and it downloads and uses the new version. Eventually the download size setting is only used for the busy-signal notification for the download status...

Reply:
It's not checked but it is used and resulted in a downlod percentage of 106% for me...

Reply:
ok, updated the size accordingly.